pinc/ProjectTransition.inc

```diff
@@ -713,7 +713,7 @@ function create_project_transitions()
             );
         }
 
-        if (1) {
+        if (1) { /** @phpstan-ignore-line */
```

Comment:
Is PHPStan ok with just `{` here without the if? I don't love that either, but I might like it better for the `if (true)` use-case.

Reply:
Yup. Done.